ush/templates/FV3LAM_wflow.xml

```diff
@@ -595,6 +693,71 @@ MODULES_RUN_TASK_FP script.
 
   </task>
 
+<!--
+************************************************************************
+************************************************************************
+-->
+  <task name="&RADAR_REFL2TTEN_TN;" cycledefs="analysis"  maxtries="{{ maxtries_radar_ref2tten }}">
```

Comment:
It will be good to introduce "refl2tten" cycleref so that this can be turned off. RTMA does not need tten. But it is fine to leave as is and a RTMA branch may add this.

Reply:
I was thinking if we need more control. Now there is a good reason to add control parameters for both non-var cloud analysis and radar_tten. Please review those controls:
DO_NONVAR_CLDANAL="FALSE"
DO_REFL2TTEN="FALSE"